devel/libnumbertext/Portfile

```diff
@@ -21,3 +21,7 @@ checksums           rmd160  31753bdd7b24c27df572867f32c19fe338b05da9 \
 
 configure.args      --disable-static \
                     --disable-werror
+
+# configure: error: Missing std::codecvt or std::regex support, use GCC 4.9 (or greater) or --enable-boost
+compiler.blacklist-append \
```

Comment:
Am I missing something and now this line works without the `compiler_blacklist_versions` portgroup?

Reply:
It is only needed when Apple clang versions are specified in blacklist.